src/examples/subsidy.ts

```diff
@@ -0,0 +1,70 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+
+import { getClient } from '~/common/client';
+import { parseArgs } from '~/common/utils';
+
+/*
+  This script showcases how to interact with Subsidy, it
+  - grants a subsidy to an account
+  - increases allowance
+  - decreases allowance
+  - sets allowance
+  - quits subsidy
+
+  Usage e.g: yarn run-example ./src/examples/subsidy.ts account=ACCOUNT_ADDRESS
+*/
+(async (): Promise<void> => {
+  const { account: beneficiary } = parseArgs<ScriptArgs>(process.argv.slice(2));
+
+  if (!beneficiary) {
+    throw new Error('Please supply an account as an argument to the script');
+  }
+
+  console.log('Connecting to the node...\n');
+
+  const api = await getClient(process.env.ACCOUNT_SEED);
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const identity = (await api.getSigningIdentity())!;
+  console.log(`Connected! Signing Identity ID: ${identity.did}`);
+
+  // get subsidizer account
+  const { account: subsidizer } = await identity.getPrimaryAccount();
+
+  // granting a subsidy to beneficary
+  const subsidyGrantQ = await api.accountManagement.subsidizeAccount({
+    beneficiary,
+    allowance: new BigNumber(1000),
+  });
+  const subsidyGrantResult = await subsidyGrantQ.run();
+
+  console.log(`Subsidy request has been created and will expire in ${subsidyGrantResult.expiry}`);
+
+  // beneficiary now has to accept the subsidy request
+
+  // get subsidy
+  const subsidy = await api.accountManagement.getSubsidy({ subsidizer, beneficiary });
+
+  // add allowance
+  const addAllowanceQ = await subsidy.increaseAllowance({ allowance: new BigNumber(1000) });
+  await addAllowanceQ.run();
+
+  // remove allowance
+  const removeAllowanceQ = await subsidy.decreaseAllowance({ allowance: new BigNumber(1000) });
+  await removeAllowanceQ.run();
+
+  // set allowance
+  const setAllowanceQ = await subsidy.setAllowance({ allowance: new BigNumber(1000) });
+  await setAllowanceQ.run();
+
+  // quit subsidy
+  const quitSubsidyQ = await subsidy.quit();
+  await quitSubsidyQ.run();
+
```

Comment:
You should also log in the value of allowance before and after modifying. Like once you get the subsidy log the current allowance, then log once after adding, removing and setting a new allowance.

Also, once you quit the subsidy maybe you can call `subsidy.exists()` and log that it doesn't exists